index.js

```diff
@@ -1456,10 +1468,23 @@ Client.prototype._fuzzing = function (synopsis) {
 
   // at the moment hard-wired to 30 minutes every 30 days
   ratio = duration / (30 * msecs.minute)
+  underscore.extend(memo, {
```

Comment:
I think that we don't need to extend it here, because it's not set to anything yet. we can just do `memo = ...`

Reply:
Also I think that we can merge this line with 1476 and 1477

Reply:
do we need this line here, because we only save it if 1478 is false

Reply:
we can do the first comment. the second one (the merge) can't be done because ratio changes, and we want both the old and new value. i can do some re-arranging on the third comment.